### PR TITLE
docs: document docs search troubleshooting

### PR DESCRIPTION
**Pull request by sozercan**
Signed-off-by: Sertac Ozercan <sozercan@gmail.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by pmengelbert — APPROVED** [review]
LGTM

File: docs/README.md [hunk 1]
```diff
@@ -39,3 +39,15 @@ $ GIT_USER=<Your GitHub username> yarn deploy
 ```
 
 If you are using GitHub pages for hosting, this command is a convenient way to build the website and push to the `gh-pages` branch.
+
+## Search
+
+Gatekeeper docs website uses Algolia DocSearch service. Please see [here](https://docusaurus.io/docs/search) for more information.
```
Comment (pmengelbert) [inline docs/README.md]: Should be Eraser, no?

**Review by pmengelbert — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (sozercan) [reply docs/README.md]: good catch!

**Review by pmengelbert — APPROVED** [review]
LGTM

**Pull request merged**